### DISCUSSION TITLE
toward J5 without B/C plugin

### DISCUSSION THREAD

**Discussion by cmirnow** [eworkers/K2ForJ4#60] [General]
I tested the latest revision of K2ForJ4 today. After upgrading to J5 and disabling the B/C plugin, I got these errors:

```
0 Class "JFactory" not found
0 Class "JPlugin" not found
```

If all K2 extensions are enabled in the extension manager, then trying to enable the B/C plugin gives this error:

`0 Class "FinderIndexerAdapter" not found`

I don't know if the attempt to adapt K2ForJ4 to Joomla 5 is complete, so I'm posting in Discussions.

**Comment by cmirnow** [comment]
Problems with _tinymce_ when editing a K2 article. The problem manifests itself regardless of enabling the plugin b / c. Other errors have gone away.

![Screenshot_20240225_231156](https://github.com/eworkers/K2ForJ4/assets/10660173/4703f75d-0f80-47cc-a155-55d80c02dc8c)

**Reply by eworkers** [reply]
doesn't seem related to K2

**Comment by eworkers** [comment]
> I tested the latest revision of K2ForJ4 today. After upgrading to J5 and disabling the B/C plugin, I got these errors:
> 
> ```
> 0 Class "JFactory" not found
> 0 Class "JPlugin" not found
> ```
> 
> If all K2 extensions are enabled in the extension manager, then trying to enable the B/C plugin gives this error:
> 
> `0 Class "FinderIndexerAdapter" not found`
> 
> I don't know if the attempt to adapt K2ForJ4 to Joomla 5 is complete, so I'm posting in Discussions.

J5 compatibility without B/C is a work in progress, although this particular issue is solved

**Comment by cmirnow** [comment]
> doesn't seem related to K2

Right. It was a weird Adguard Anti-Banner glitch....

**Comment by eworkers** [comment]
It safe to say that K2ForJ4 is joomla 5 compatible without enabling B/C plugin (=> J5 fully compatible)
